http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -299,6 +303,13 @@ protected void configure(SelectableChannel selectable)
 
         httpClient.setConnectBlocking(config.isConnectBlocking());
 
+        if (mbeanServer.isPresent()) {
+            // export jmx mbeans if a server was provided
+            MBeanContainer mbeanContainer = new MBeanContainer(mbeanServer.orElseThrow());
+            mbeanContainer.setDomain("org.eclipse.jetty.http.client." + name);
```

Comment:
Where does `"org.eclipse.jetty.http.client." + name` show up? How does it affect the mbean names?

Reply:
Alternative is: 

<img width="696" height="660" alt="CleanShot 2025-09-09 at 12 35 37@2x" src="https://github.com/user-attachments/assets/f6f083d8-41be-4a99-bf3a-6a2ab5826825" />

Reply:
I'm checking whether I can name a client (instead of `oejc_HttpClient@identityHashCode`)